zerver/actions/pinned_views.py

```diff
@@ -0,0 +1,83 @@
+from typing import Any
+
+from django.db import transaction
+from django.utils.translation import gettext as _
+
+from zerver.lib.exceptions import JsonableError, ResourceNotFoundError
+from zerver.models import PinnedView, UserProfile
+from zerver.models.pinned_views import LeftSidebarViewLocationEnum
+from zerver.tornado.django_api import send_event_on_commit
+
+VALID_VIEW_IDS = ["inbox", "recent", "reactions", "mentions", "feed", "drafts", "starred"]
```

Comment:
Let's call this `BUILT_IN_VIEW_IDS`; then we won't have to rename this in the future when we allow custom options here.

Also might as well do one per line.